pkg/reflector/reflector.go

```diff
@@ -280,6 +293,43 @@ func ReflectorFromConfig(config ReflectorConfig) (*Reflector, error) {
 	}, nil
 }
 
+func emitMetricFromFile() error {
+	name := "/var/spool/ctlstore/metrics.json"
```

Comment:
let us os.Stat to check if the file exists. If it doesn't, we should return nil since its not an error